src/state/routing/slice.ts

```diff
@@ -166,10 +166,9 @@ export const routingApi = createApi({
                   typeof errorData === 'object' &&
                   (errorData?.errorCode === 'NO_ROUTE' || errorData?.detail === 'No quotes available')
                 ) {
-                  Sentry.withScope((scope) => {
-                    scope.setExtra('requestBody', requestBody)
-                    scope.setExtra('response', response)
-                    Sentry.captureException(new Error("No routes found for user's quote request, alert Routing Team"))
+                  sendAnalyticsEvent('No quote received from routing API', {
+                    requestBody,
+                    response,
```

Comment:
maybe log the router preference here too? Would be good to distinguish between quote lookups and price lookups